src/pages/AdminList/AdminList.js

```diff
@@ -30,3 +31,5 @@ export default function AdminList() {
     </Box>
   );
 }
+
+export default withRouter(AdminList);
```

Comment:
you're not using the router props in the component so you don't need with router HOC